src/BoundaryConditions/fill_halo_regions.jl

```diff
@@ -45,10 +45,13 @@ end
 const MaybeTupledData = Union{OffsetArray, NTuple{<:Any, OffsetArray}}
 
 "Fill halo regions in ``x``, ``y``, and ``z`` for a given field's data."
-function fill_halo_regions!(c::MaybeTupledData, boundary_conditions, indices, loc, grid, args...; kwargs...)
-
+function fill_halo_regions!(c::MaybeTupledData, boundary_conditions, indices, loc, grid, args...; fill_boundary_normal_velocities = true, kwargs...)
```

Comment:
this line is too long, split into multiple lines

Reply:
I've done this now but thought I should check, there is no clear rules on how lines are split right?